test/checks.js

```diff
@@ -27,6 +29,30 @@ describe('checks', function () {
       });
     });
 
+    var idempotencyKey = uuid();
+
+    it('creates a check with an idempotency key', function (done) {
+      Lob.checks.create(CHECK, {
+        'idempotency-key': idempotencyKey
```

Comment:
I don't think so. I think this is how node sets headers, I believe this is correct